manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MatchTest.scala

```diff
@@ -245,7 +245,21 @@ class MatchTest extends DocumentingTest {
               | you define a single link of a path -- the starting node, the connecting relationship and the end node. Characteristics describing the relationship
               | like relationship type, max hops and direction are all used when finding the shortest path.
               | If there is a WHERE clause following the match of a shortestPath, relevant predicates will be included in the shortestPath.
-              | If the predicate is a NONE() or ALL() on the relationship elements of the path, it will be used during the search to improve performance.""")
+              | If the predicate is a +NONE()+ or +ALL()+ on the relationship elements of the path, it will be used during the search to improve performance (see <<query-shortestpath-planning>>).""")
```

Comment:
I think the docs guys are working on conventions for the manual code, and whether to use + or ` signs to get `code`-style font, and I _think_ they are leaning towards ` over +. @nawroth @jjaderberg?